scripts/common.ts

```diff
@@ -57,3 +57,7 @@ export function getTKey(arg: Node): string | null {
     }
     return null;
 }
+
+export function isTranslation(translation: Translations[string]): translation is PluralisedTranslation {
```

Comment:
shouldn't this be `isPluralisedTranslation`? We know it is a translation.

Reply:
Yup, my bad